fe/fe-core/src/main/java/com/starrocks/planner/EsScanNode.java

```diff
@@ -237,23 +239,38 @@ protected String getNodeExplainString(String prefix, TExplainLevel detailLevel)
             output.append(prefix).append("SORT COLUMN: ").append(sortColumn).append("\n");
         }
 
-        if (!conjuncts.isEmpty()) {
+        if (conjuncts.isEmpty()) {
             output.append(prefix).append("PREDICATES: ").append(
                     getExplainString(conjuncts)).append("\n");
-            // reserved for later using: LOCAL_PREDICATES is processed by StarRocks EsScanNode
-            output.append(prefix).append("LOCAL_PREDICATES: ").append(" ").append("\n");
-            // reserved for later using: REMOTE_PREDICATES is processed by remote ES Cluster
-            output.append(prefix).append("REMOTE_PREDICATES: ").append(" ").append("\n");
-            // reserved for later using: translate predicates to ES queryDSL
-            output.append(prefix).append("ES_QUERY_DSL: ").append(" ").append("\n");
-        } else {
             output.append(prefix).append("ES_QUERY_DSL: ").append("{\"match_all\": {}}").append("\n");
+        } else {
+            QueryConverter queryConverter = new QueryConverter();
+            QueryBuilders.QueryBuilder queryBuilder = queryConverter.convert(getConjuncts());
+            output.append(prefix).append("PREDICATES: ").append(
+                    getExplainString(conjuncts)).append("\n");
+            // reserved for later using: LOCAL_PREDICATES is processed by StarRocks EsScanNode
+            output.append(prefix).append("LOCAL_PREDICATES: ")
+                    .append(getExplainString(queryConverter.localConjuncts()))
+                    .append("\n");
+            output.append(prefix).append("REMOTE_PREDICATES: ")
```

Comment:
Shall we unify the naming style? Such as `Remote Predicates: `?

Reply:
Are there any other places? I didn't notice this